libtransmission/magnet-metainfo.h

```diff
@@ -22,11 +22,12 @@ class tr_magnet_metainfo
     bool parseMagnet(std::string_view magnet_link, tr_error** error = nullptr);
 
     std::string magnet() const;
-
+    
```

Comment:
_(nit)_ accidentally-introduced whitespace, here and a few lines down on line 30

Reply:
Yup I noticed tgis